templates/healthBar/game.json

```diff
@@ -0,0 +1,11558 @@
+{
+  "firstLayout": "",
+  "gdVersion": {
+    "build": 99,
+    "major": 4,
+    "minor": 0,
+    "revision": 0
+  },
+  "properties": {
+    "adaptGameResolutionAtRuntime": true,
+    "folderProject": false,
+    "orientation": "landscape",
+    "packageName": "com.example.HealthExample",
+    "pixelsRounding": true,
+    "projectUuid": "5aaac4f4-df86-47fe-94e9-3d8b81cf3839",
+    "scaleMode": "nearest",
+    "sizeOnStartupMode": "",
+    "templateSlug": "",
+    "useExternalSourceFiles": false,
+    "version": "1.0.0",
+    "name": "HealthExample",
```

Comment:
This will be the name creators will have for their project (and if they publish it right way). Maybe rename it to "Health Bar Lesson" to match the parallax one ("Camera Parallax Lesson")